platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -263,18 +273,33 @@ private MigrationReport doImport(Path exportDirectory,
         }
         report.end();
         if (report.hasErrors()) {
+            SecurityLogger.audit("Errors importing configuration settings from file {}",
+                    exportFile);
             report.record(new MigrationException(Messages.IMPORT_FAILURE, exportFile));
         } else if (report.hasWarnings()) {
+            SecurityLogger.audit("Warnings importing configuration settings from file {}",
+                    exportFile);
+            // don't leave the zip file there if the import succeeded
+            SecurityLogger.audit("Deleting exported file {}", exportFile);
+            FileUtils.deleteQuietly(exportFile.toFile());
             report.record(new MigrationWarning(Messages.IMPORT_SUCCESS_WITH_WARNINGS, exportFile));
             report.record(new MigrationWarning(Messages.RESTART_SYSTEM_WHEN_WARNINGS));
         } else {
+            SecurityLogger.audit("Success importing configuration settings from file {}",
+                    exportFile);
+            // don't leave the zip file there if the import succeeded
+            SecurityLogger.audit("Deleting exported file {}", exportFile);
```

Comment:
❓ Not a big deal, but do you think deleteQuietly would ever fail and leave the zip behind?

Reply:
If it fails then there is not much we can do. Should we audit the failure too? @stustison?

Reply:
Usually with audits you would audit just the concrete actions. "I deleted XYZ" "I did not delete XYZ" you would not audit specific errors that crop up or whether or not you're "trying" to do something, just that you either did it, or you didn't.

Reply:
Made changed based on our discussion